common/views/components/NewsletterPromo/NewsletterPromo.tsx

```diff
@@ -206,18 +204,11 @@ const NewsletterPromo = () => {
                       onSubmit={handleSubmit}
                       noValidate={isEnhanced}
                     >
-                      <input type="hidden" name="userid" value="225683" />
                       <input
                         type="hidden"
-                        name="SIG22a9ece3ebe9b2e10e328f234fd10b3f5686b9f4d45f628f08852417032dc990"
```

Comment:
These are values you need if you use the Dotdigital-generated HTML sign-up form, but we're calling our own API endpoint here – so these are unused and misleading.

Reply:
hmm on review of dotdigital documentation i realise campaign and addressbook are different things - ignore me